app.js

```diff
@@ -19,8 +19,9 @@ mongoose.connect(config.MONGODB_URI, { useNewUrlParser: true, useUnifiedTopology
     logger.error('error connection to MongoDB:', error.message)
   })
 
+app.use('/',express.static('build')) // look in the build directory to serve frontend
```

Comment:
Why do we need to indicate the root directory?

Reply:
It means "when we access the root domain" (i.e. sparkorbital.herokuapp.com), it will look inside /build folder and serve the index.js (which is our frontend app)